modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/TcpCommunicationMetricsListener.java

```diff
@@ -121,11 +134,17 @@ public TcpCommunicationMetricsListener(GridMetricManager mmgr) {
         );
 
         sentMsgsCntByNodeIdMetricFactory = nodeId ->
```

Comment:
Why we should keep this metric if it totally replaced with the new one?

Reply:
old JMX API

Reply:
@ibessonov It seems there are no release with metrics by node. Old metric shows total sent/received message count.